charts/kube-prometheus-stack/templates/alertmanager/servicemonitor.yaml

```diff
@@ -52,7 +52,9 @@ spec:
     {{- if .Values.alertmanager.serviceMonitor.proxyUrl }}
     proxyUrl: {{ .Values.alertmanager.serviceMonitor.proxyUrl}}
     {{- end }}
-    scheme: http
+    {{- if .Values.alertmanager.serviceMonitor.scheme }}
+    scheme: {{ .Values.alertmanager.serviceMonitor.scheme }}
```

Comment:
`https` scheme may require `tlsConfig` to work. Refer to example ServiceMonitor in https://github.com/prometheus-community/helm-charts/pull/4151/files#r1461352647.

Reply:
@DrFaust92 could you please add tlsConfig?

Reply:
added